Gemfile

```diff
@@ -1,3 +1,6 @@
 source 'https://rubygems.org'
 gemspec
 
+if RUBY_VERSION < '2.3'
```

Comment:
Hey @andrehjr, thanks for the PR. Maybe we could lock `rexml` for `latest_ruby`? https://github.com/kyrylo/latest_ruby

The more accurate way to compare versions would be:

```rb
if Gem::Version.new(RUBY_VERSION) < Gem::Version.new('2.3')
```

Reply:
True, I updated the PR! Thanks for pointing that! 

I'm thinking if there is a way to lock rexml on latest_ruby without much trouble:
1) Release a version on latest_ruby locking rexml < 3.2.5 to be used for Ruby < 2.3. As bundle install will try to pull the latest version as Gemfile.lock is on .gitignore. We would still need to specify the version here somehow? Or have a custom Gemfile.lock for Ruby <2.3.

Another alternative:
2) https://github.com/pry/pry-doc/blob/master/Rakefile#L12 remove this require, and rework the Rakefile to only require latest_ruby when it's actually needed. The issue is that whoever runs pry-doc repo on Ruby < 2.3 in order to generate docs would need to install a working rexml version on their own.

Or 3) rework the following part on latest_ruby to not use rexml https://github.com/kyrylo/latest_ruby/blob/983518b6f1c83abb51840d6bd714596b887db90a/lib/latest_ruby/retrievers/rubinius_retriever.rb#L1 Just making that 'require' breaks stuff on Ruby 2.2. Maybe this is the best way to deal with issue and fix it there instead?

I'm cool with either approach and can make the necessary changes :)  WDYT? Am I missing something?

Thanks for review!

Reply:
Yes! 🙌 

Rebuilding the 3.0.0 branch here should have all builds green now :)